docker/Dockerfile

```diff
@@ -175,10 +182,12 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     ca-certificates
 
-# Config settings
-
+# Config settings for zebrad
 ARG NETWORK
-ENV NETWORK=${NETWORK:-Mainnet}
+ENV NETWORK=${NETWORK}
+
+ARG FEATURES
+ENV FEATURES=${FEATURES}
```

Comment:
```suggestion
ENV FEATURES=${FEATURES}

ARG RPC_PORT
ENV RPC_PORT=${RPC_PORT}
```

I think we need this to pass the port from here https://github.com/zcashfoundation/zebra/blob/3944c3f199e31ef2e992cd31bc67d7d08d9c124b/.github/workflows/release-binaries.yml#L46 to `runtime-entrypoint.sh`

Reply:
The `$RPC_PORT` is just used as an env variable at runtime, we'd only need an `ARG` here if this required to be passed at buildtime, but it's not.

Reply:
I don't think that's right for the mining image. The currently released `.experimental` image doesn't work because it has no RPC port specified. You can see it yourself if you run the container and look at the config.

Reply:
The docs say that `-n string` is analogous to just `string`: 
![image](https://github.com/ZcashFoundation/zebra/assets/8863307/d0256c47-6f3a-430e-9de0-d3ff4df7a929)

so adding `-n` shouldn't do anything?

I must be missing something. How will the port propagate from https://github.com/zcashfoundation/zebra/blob/3944c3f199e31ef2e992cd31bc67d7d08d9c124b/.github/workflows/release-binaries.yml#L46 to `runtime-entrypoint.sh`?

Another solution would be to remove https://github.com/zcashfoundation/zebra/blob/3944c3f199e31ef2e992cd31bc67d7d08d9c124b/.github/workflows/release-binaries.yml#L46 and https://github.com/ZcashFoundation/zebra/blob/ee54a59ee20fcae73a04f26198146b23ccd0de39/.github/workflows/build-docker-image.yml#L166, and let `runtime-entrypoint.sh` handle the port based only on `getblocktemplate-rpcs` in `$FEATURES`.

Reply:
> The docs say that `-n string` is analogous to just `string`

That's correct, maybe it's because we're using double quotes when passing the argument
- https://github.com/ZcashFoundation/zebra/pull/7200#issuecomment-1634653085
- https://github.com/ZcashFoundation/zebra/pull/7179/files#diff-573d08e7251621753237189e973a77b6c83e29e9b4b5f16d3f8fc8f26ed053b3R22

The following `ARGs` should not exist in the `release-binaries.yml` workflow:
- network
- rpc_port
- test_features
- zebra_skip_ipv6_tests

I'd even remove others. Those changes will follow on a PR depending on this one. I explained why those are not needed here: https://github.com/ZcashFoundation/zebra/pull/7200#discussion_r1263572502 but I'm not sure if that's clear.

Reply:
I see now. Shouldn't we then remove https://github.com/ZcashFoundation/zebra/blob/7b0dedd3a28b3bcb24214ecae7d00408e5349749/.github/workflows/build-docker-image.yml#L166 as well?

Reply:
Yes, we should also remove that.

Reply:
All clear now. Thanks, Gustavo.

I think we can even remove the `-n`.